tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38,py39
+envlist = py39,py311
```

Comment:
It would be ideal to add py310 here also; I suppose if 3.11 works 3.10 should also work

Reply:
Good idea. Fixed.